drivers/shared/executor/executor_linux.go

```diff
@@ -161,7 +161,10 @@ func (l *LibcontainerExecutor) Launch(command *ExecCommand) (*ProcessState, erro
 	if err != nil {
 		return nil, fmt.Errorf("failed to determine relative path base=%q target=%q: %v", command.TaskDir, path, err)
 	}
-	path = rel
+
+	// Turn relative-to-chroot path into absolute path to avoid
+	// libcontainer trying to resolve the binary using $PATH
+	path = "/" + rel
```

Comment:
We probably should check here if `rel` starts with `../` and report an error with binary being outside libcontainer root.